docker/docker.hcl

```diff
@@ -0,0 +1,23 @@
+group "default" {
+  targets = ["graphql-mesh"]
+}
+
+variable "PWD" {
+  default = "."
+}
+
+variable "context" {
+  default = "${PWD}/docker"
+}
+
+target "graphql-mesh" {
+  context = "${context}"
+  dockerfile = "${PWD}/docker/Dockerfile"
+  platforms = ["linux/amd64", "linux/arm64"]
+  tags = [
+    "ghcr.io/ardatan/graphql-mesh:v1"
+  ]
+  args = {
```

Comment:
This one is not needed, and might leak some tokens?

Reply:
yes, correct, my bad, that was one of the things I pushed while debugging HCL issues